frontend/src/pages/account/reverseShares.tsx

```diff
@@ -119,19 +123,19 @@ const MyShares = () => {
                           <Accordion.Panel>
                             {reverseShare.shares.map((share) => (
                               <Group key={share.id} mb={4}>
-                                <Text maw={120} truncate>
-                                  {share.id}
-                                </Text>
+                                <Anchor href={`${appUrl}/share/${share.id}`}>
```

Comment:
I think it would be better that the link opens in a new tab. What do you think?

Reply:
Yes indeed